test/Engine/ProtoTest/Associative/MicroFeatureTests.cs

```diff
@@ -3169,6 +3169,17 @@ public void TestNullsOnExpression01()
             thisTest.Verify("b", false);
         }
 
+        [Test]
+        public void TestAddNullToString()
+        {
+            string code =
+@"
+a = ""hello"" + null;
+";
+            thisTest.RunScriptSource(code);
+            thisTest.Verify("a", null);
+        }
+        
 

```

Comment:
Git issues in diff. Here is the added test:

 [Test]
        public void TestAddNullToString()
        {
            string code =
@"
a = ""hello"" + null;
";
            thisTest.RunScriptSource(code);
            thisTest.Verify("a", null);
        }